bin/MorphingMSSMvsSM.cpp

```diff
@@ -2077,9 +2077,20 @@ int main(int argc, char **argv) {
 
   for(auto u : met_uncerts) ConvertShapesToLnN (cb.cp().bin_id(mssm_bins).process({"ZTT"}, false), u);
 
+  if(variable=="m_sv_puppi" || variable=="m_sv_VS_pt_tt_splitpT") {
+    // convert ggH theory uncertainties to lnN when fitting m_sv
+    std::vector<std::string> ggh_theory = {"Hdamp_ggH_t_REWEIGHT","Hdamp_ggH_b_REWEIGHT","Hdamp_ggH_i_REWEIGHT","QCDscale_ggH_REWEIGHT"}; 
+    for(auto u : ggh_theory) ConvertShapesToLnN (cb.cp().bin_id(mssm_bins).process({"ZTT"}, false), u);
```

Comment:
The process filter is unnecessary here but this does not make a difference for the datacards. It is thus fine to leave it in.